doc/content/concepts/events.md

```diff
@@ -0,0 +1,209 @@
+---
```

Comment:
Is this file in the right place? This was part of the Getting Started, right?

Reply:
moved it back

Reply:
?

Reply:
The events section, which is pretty advanced, will not be in getting started anymore